server/src/repositories/prisma/PrismaUsersRepository.ts

```diff
@@ -0,0 +1,28 @@
+import { IUsersRepositories } from '@repository/IUsersRepositories'
+import { prisma } from '../../database/client'
+import { User } from '@entities/User'
+
+class PrismaUsersRepository implements IUsersRepositories {
```

Comment:
Why prefix Prisma and Not only UserRepository?

Reply:
Using interface as contract, if you want to change database or use another type of database manager, just change the implementation

Reply:
Know type of implementation

Reply:
Do you think it's unnecessary?

Reply:
Umh! No, is a good pattern